lms/djangoapps/mobile_api/video_outlines/tests.py

```diff
@@ -149,22 +196,49 @@ def test_chapter_is_none(self):
         section_url = course_outline[0]["section_url"]
         unit_url = course_outline[0]["unit_url"]
         self.assertRegexpMatches(section_url, r'courseware$')
-        self.assertTrue(section_url)
-        self.assertTrue(unit_url)
         self.assertEqual(section_url, unit_url)
 
-    def test_section_is_none(self):
+        path = course_outline[0]['path']
+        self.assertEqual(0, len(path))
+        #Named path will be deprecated eventually
+        named_path = course_outline[0]['named_path']
+        self.assertEqual(0, len(named_path))
+
+    def test_video_under_course(self):
         """
-        Tests when there is no section under chapter, and video under chapter
+        Tests when there is a video under vertical directly under course
         """
         self.login_and_enroll()
-        self.chapter = ItemFactory.create(  # pylint:disable=W0201
-            parent_location=self.course.location,
-            category="chapter",
-            display_name=u"test factory chapter omega \u03a9",
+        ItemFactory.create(
+            parent_location=self.vertical_under_course.location,
+            category="video",
+            display_name=u"test factory video omega \u03a9",
+        )
```

Comment:
Oh.  I now see in the docstring that this is supposed to test a video under a vertical under the course.

To make this easier to understand, here are 2 suggestions.  Either:
*Option A:* Rename the test methods to be explicit about the hierarchy.  For example:
```
test_under_course
test_under_course_chapt
test_under_course_vert
test_under_course_chapt_sec
test_under_course_chapt_sec_vert
etc.
```

*Option B:* Building on @aleffert's earlier suggestion, consider just having one test that creates the structure of the course and adds a video to each component of the course (under course, under chapter, under section, under chapter/section/vertical, under chapter/vertical, under section/vertical, etc.).  And then the API should return each video with the corresponding correct position.